.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+on:
+  create:
+    tags:
+       - 'v*.*'
```

Comment:
I'd like to go the full three-level-deep [semantic versioning](https://semver.org/) approach here. So things like `v0.0.1` should pass. Could you also make sure that it gets triggered by things like `v0.0.1-RC1` so we can create Release Candidates through this process?

Reply:
Done, but formal digit based regex to come.

Reply:
See other conversation for implementation.